hail/python/hailtop/batch/backend.py

```diff
@@ -53,24 +54,16 @@ class Backend(abc.ABC, Generic[RunningBatchType]):
     """
     _closed = False
 
-    def __init__(self):
-        self._requester_pays_fses: Dict[GCSRequesterPaysConfiguration, RouterAsyncFS] = {}
+    def __init__(self, requester_pays_fses: GCSRequesterPaysFSCache):
+        import nest_asyncio  # pylint: disable=import-outside-toplevel
+        nest_asyncio.apply()
```

Comment:
I think this `nest_asyncio` was deleted in main. Is this still necessary?

Reply:
ooo good catch, thank you!